packages/python/plotly/js/widget.ts

```diff
@@ -1679,7 +1678,7 @@ function py2js_deserializer(v: any, widgetManager?: any) {
     for (var i = 0; i < v.length; i++) {
       res[i] = py2js_deserializer(v[i]);
     }
-  } else if (_.isPlainObject(v)) {
+  } else if (_.isObject(v)) {
```

Comment:
Personally, I'd just make my own helper for this. Not sure how much lodash is used in the rest of the file, but a nice typescript helper i often copy into projects:

```ts
function isObject(input: unknown): input is Record<string, unknown> {
  return typeof input === 'object' && input !== null && !Array.isArray(input);
};
```

Reply:
@marthacryan typedArrays should also be excluded.
```js
var ab = ArrayBuffer;
var dv = DataView;

function isTypedArray(a) {
    return ab.isView(a) && !(a instanceof dv);
}
```